primitives/src/policy.rs

```diff
@@ -493,31 +486,31 @@ impl Policy {
 
     /// Returns the supply at a given time (as Unix time) in Lunas (1 NIM = 100,000 Lunas). It is
     /// calculated using the following formula:
-    /// Supply (t) = Genesis_supply + Initial_supply_velocity / Supply_decay * (1 - 2^(- Supply_decay * t))
-    /// Where t is the time in milliseconds since the PoW genesis block and `genesis_supply` is the supply at
+    /// ```text
+    /// supply(t) = total_supply - (total_supply - genesis_supply) * supply_decay^t
+    /// ```
+    /// Where t is the time in milliseconds since the PoS genesis block and `genesis_supply` is the supply at
     /// the genesis of the Nimiq 2.0 chain.
     #[cfg_attr(feature = "ts-types", wasm_bindgen(js_name = supplyAt))]
-    pub fn supply_at(genesis_supply: u64, current_time: u64) -> u64 {
-        assert!(current_time >= Self::POW_GENESIS_TIMESTAMP);
-
-        let t = current_time - Self::POW_GENESIS_TIMESTAMP;
```

Comment:
You can remove the `POW_GENESIS_TIMESTAMP` constant now.

Reply:
Done.

Reply:
It's used in other places in the code though, so this will now lead to a compile error.